.github/workflows/pypi.yml

```
@@ -0,0 +1,40 @@
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+
+      - name: Install dependencies
+        run: poetry install
+
+      - name: Build package (optional)
+        run: poetry build
+      
+      - name: install twine
+        run: pip install twine
+      
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)